Dockerfile

```diff
@@ -2,4 +2,8 @@ FROM jenkins/jenkins:2.471-jdk17
 
 COPY logos /usr/share/jenkins/ref/userContent/logos
 COPY ./plugins.txt /usr/share/jenkins/ref/plugins.txt
-RUN jenkins-plugin-cli -f /usr/share/jenkins/ref/plugins.txt --verbose
+RUN jenkins-plugin-cli \
+  --jenkins-update-center='https://azure.updates.jenkins.io/update-center.json' \
+  --jenkins-plugin-info='https://azure.updates.jenkins.io/update-center.json' \
```

Comment:
why are these set to the same files? they aren't the same thing, plugin-info normally goes to https://updates.jenkins.io/current/plugin-versions.json

on the new UC what maps to https://azure.updates.jenkins.io/current/plugin-versions.json which seems to work fine?

Reply:
Fixup in #1661, many thanks @timja !